execute/observation.go

```diff
@@ -124,11 +124,11 @@ func (p *Plugin) Observation(
 	return p.ocrTypeCodec.EncodeObservation(observation)
 }
 
-func (p *Plugin) getCurseInfo(ctx context.Context, lggr logger.Logger) (*reader.CurseInfo, error) {
+func (p *Plugin) getCurseInfo(ctx context.Context, lggr logger.Logger) (reader.CurseInfo, error) {
```

Comment:
Switching to value type just a preference / style thing or does that help somehow?

Reply:
It's taken from the cache and value type gives us "copy" instead of relying on reference to something that could be wiped out in the meantime. It's just safer

Reply:
Ah I see, got it, makes sense.